src/DynamoCoreWpf/ViewModels/Preview/WatchViewModel.cs

```diff
@@ -139,9 +144,46 @@ public bool IsOneRowContent
             }
         }
 
+
+        /// <summary>
+        /// Number of items in the overall list if node output is a list
+        /// </summary>
+        public int NumberOfItemsWT
+        { 
+            get { return numberOfItems; }
```

Comment:
`numberOfItems` is static member, so the property should be static as well. Or, change `numberOfItems` to be a instance field.